javascript/javascript1/week1/age-ify.js

```diff
@@ -0,0 +1,6 @@
+const yearOfBirth = 1992;
+let yearFuture = 2045;
+console.log("typeof the variable 'yearOfBirth' is: " + typeof(yearOfBirth));
+console.log("typeof the variable 'yearFuture' is: " + typeof(yearFuture));
+let age = yearFuture - yearOfBirth;
+console.log("you will be " + age + " years old in " + yearFuture);
```

Comment:
just another way to combine the string and variable (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals). i find it easy just a suggestion.